src/main/java/wromaciej/moviesrental/movies/controllers/MoviesController.java

```diff
@@ -34,6 +37,11 @@ public MoviesController(MovieService movieService, CartService cartService) {
         return cartService.calculateRentalRateForDays(movieId, days);
     }
 
+    @RequestMapping(value="/{movieId}/actors")
+    public @ResponseBody Collection<Actor> findMoviesWithActor(@PathVariable int movieId) {
```

Comment:
method name is wrong. findMoviesWithActor would indicate you pass actorId and get all movies the actor starred in.

Reply:
done